core/src/providers/anthropic/anthropic.rs

```diff
@@ -326,6 +329,10 @@ impl LLM for AnthropicLLM {
         }
     }
 
+    fn set_tokenizer_from_config(&mut self, config: crate::types::tokenizer::TokenizerConfig) {
```

Comment:
Can you do a top-level `use` so we can use the shorthand here (just `TokenizerConfig instead of crate::...) 🙏